### DISCUSSION TITLE
NumericUpDown Validation error

### DISCUSSION THREAD

**Discussion by Amenti** [AvaloniaUI/Avalonia#16914] [Q&A] [answered]
On iOS and Android when emptying a NumericUpDown via the keyboard I see this validation adorner with 11.1.3. This did not happen in older Avalonia-versions and I'd like to deactivate the notice. Removing the DataValidator-Adordners seems not to work. What could I do?

<img width="257" alt="Bildschirmfoto 2024-09-04 um 07 25 40" src="https://github.com/user-attachments/assets/ae9dac2c-3fa7-4a5c-ac42-46bc5b3a44d5">

**Comment by workgroupengineering** [comment, marked as answer]
You make Target Property Nullable or set in Binding TargetNullValue or  Custom Converter

**Reply by Amenti** [reply]
Thank you. TargetNullValue did not make a difference but setting the FallBackValue or "nullabling" the property worked.
